src/web/app/pages-instructor/instructor-student-activity-logs/instructor-student-activity-logs.component.html

```diff
@@ -0,0 +1,123 @@
+<h1>Student Activity Logs For {{course.courseName}}</h1>
+<p>This page allows you to find when your students have accessed or submitted a particular feedback session for a given course. Note that you can only view student logs if you have <b>owner or manager privileges</b> for the course. The earliest date you can search for is <b>{{this.LOGS_RETENTION_PERIOD}} days</b> before today.</p>
```

Comment:
The description is no longer accurate:
- `accessed or submitted a particular feedback session for a given course`
  - There is another possible action not mentioned here.
  - "given course" should be improved, or just not mentioned at all. It is known exactly which course is being involved here.
- `The earliest date you can search for is <b>{{this.LOGS_RETENTION_PERIOD}} days</b> before today.`
  - Not necessarily true once the backend is reworked, so do remember to update this when the time comes.